app/src/test/java/com/futurice/freesound/network/api/DefaultFreeSoundSearchServiceTest.java

```diff
@@ -62,11 +61,10 @@ public void setUp() {
     public void search_emitsResults_whenApiSuccessful() {
         new Arrangement().withApiResultResult(RESULT);
 
-        TestSubscriber<SoundSearchResult> ts =
-                defaultFreeSoundSearchService.search(QUERY).test();
-
-        ts.assertNoErrors()
-          .assertValue(RESULT);
+        defaultFreeSoundSearchService.search(QUERY)
+                                     .test()
+                                     .assertNoErrors()
```

Comment:
Exactly, now they would put something like that:
`java.lang.AssertionError: Expected: 1 (class: Integer), Actual: [] (latch = 0, values = 0, errors = 1, completions = 0)`